app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict[any]], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy > 0:
```

Comment:
You can use:

        if masks_to_buy:
            return f"Friends should buy {masks_to_buy} masks"
            
instead of 

    if masks_to_buy > 0:
        return f"Friends should buy {masks_to_buy} masks"

Reply:
thx I miss about it ;)